packages/xstate-react/src/fsm.ts

```diff
@@ -9,6 +9,22 @@ import {
 import { useSubscription, Subscription } from 'use-subscription';
 import useConstant from './useConstant';
 
+const getServiceState = <
+  TContext extends object,
+  TEvent extends EventObject = EventObject,
+  TState extends Typestate<TContext> = any
+>(
```

Comment:
Maybe we should "expose" `service.state` just like in XState core?

Reply:
Maybe, this can be easily added and would make this use case easier, just didn't want to add this on my own in the spirit of keeping `@xstate/fsm` minimal.